### ISSUE TITLE
seissol does not compile anymore on supermuc

### ISSUE THREAD

**Issue by Thomas-Ulrich** [SeisSol/SeisSol#1065]
**Describe the bug**
seissol does not compile on supermuc:

```
[ 52%] Building CXX object CMakeFiles/SeisSol-lib.dir/src/Solver/time_stepping/AbstractTimeCluster.cpp.o
/hppfs/work/pr45fi/di73yeq4/SeisSol/src/ResultWriter/EnergyOutput.cpp(284): internal error: null pointer
  }
  ^
```
but 
v1.1.3 can be compiled.

**To Reproduce**

```
Currently Loaded Modulefiles:
 1) admin/1.0     3) lrz/1.0        5) intel-oneapi-compilers/2021.4.0(default:intel)   7) intel-mpi/2019-intel   9) cmake/3.21.4            11) numactl/2.0.14-intel21                
 2) tempdir/1.0   4) spack/22.2.1   6) intel-mkl/2020                                   8) gcc/11.2.0            10) python/3.8.11-extended  12) seissol-env/develop-intel21-impi-x2b  
```
commit fe89015d7b200e5683583018951cfda7ed529101

```
cmake -DNUMA_AWARE_PINNING=ON -DASAGI=ON -DCMAKE_BUILD_TYPE=Release -DHOST_ARCH=skx -DPRECISION=double -DORDER=4 -DGEMM_TOOLS_LIST=LIBXSMM,PSpaMM ..
```

```
di73yeq4@login02:/hppfs/work/pr45fi/di73yeq4/SeisSol/build-v1.1.3> mpicc --version
icc (ICC) 2021.4.0 20210910
Copyright (C) 1985-2021 Intel Corporation.  All rights reserved.
```

**Comment by sebwolf-de** [comment]
Hi, I can reproduce the error.

**Comment by sebwolf-de** [comment]
The problem was the `min` reduction over a member variable....

**Comment by sebwolf-de** [comment]
Fixed with #1067
